setup.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
Why do we need all this setup.py, setup.cfg. tox etc stuff? I suppose it's only for libs.